common/lib/xmodule/xmodule/js/src/video/05_video_quality_control.js

```diff
@@ -10,18 +10,16 @@ function () {
         '<button class="control quality-control is-hidden" aria-disabled="false" title="',
             gettext('High Definition'),
         '">',
-            '<span class="icon-fallback-img">',
-                '<span class="icon icon-hd" aria-hidden="true">HD</span>', // "HD" is treated as a proper noun
+            '<span class="icon icon-hd" aria-hidden="true">HD</span>', // "HD" is treated as a proper noun
+            // Translator note:
+            // HD stands for high definition
```

Comment:
Looks like this translator comment can be removed-- HD isn't being used for the sr text.